style.css

```diff
@@ -570,7 +570,8 @@ form {
 
   /* Add background PNGs to contact form */
   #contact-section {
-    background-image: url("img/desktop-bg/contact-form-illustration1.png"),
+    background-image:
+      url("img/desktop-bg/contact-form-illustration1.png"),
       url("img/desktop-bg/contact-form-illustration2.png"),
       url("img/desktop-bg/contact-form-illustration3.png");
     background-repeat: no-repeat;
```

Comment:
- [ ] (OPTIONAL) Good job so far. Kindly extend your form inputs so that it matches the template design. Right now your form inputs are short.
- [ ] (OPTIONAL) It would also be good to apply the correct background images to your contact section for a better user interface 👍 
- Your design:
![Annotation 2022-03-26 072922](https://user-images.githubusercontent.com/85113567/160227792-ca1a81fe-290d-4230-b311-e3be5bd2976a.jpg)
- Figma design:
![Annotation 2022-03-26 072906](https://user-images.githubusercontent.com/85113567/160227802-88888799-632b-4fb7-82f3-47af549f2787.jpg)

Reply:
Done, Thank you for the observation.